application/locales/global.en.yaml

```diff
@@ -143,34 +143,34 @@
 
 # Request Post Status Transition email subjects
 - id: Email.Subject.Request.FromAcceptedToCommitted
-  translation: Oops, you are not yet expected to fulfill a certain {{.AppName}} request
+  translation: Your {{.AppName}} offer for "{{.postTitle}}" may no longer be needed
```

Comment:
On these should `{{.postTitle}}` go at the end in case it's long? Also, are there character count limits on subjects?

Reply:
I was thinking about in the job portion if we should only send a substring of the post title, but I didn't make that change yet

Reply:
Post title is too long (255) in the database. It would help some to reduce that.

Reply:
Looks like I can see over 70 characters of a subject line using default density in gmail on my laptop screen. I think I'll truncate the post title to just 16 characters, that should be more than enough to identify which request it is, without needing a ton of detail. How does that sound?

Reply:
Do you want to add ` ...` after the 16 characters if the Title is longer than that?

Reply:
Yep, that's what it'll do now.